WebCrawler/obj/Debug/App.g.cs

```diff
@@ -1,4 +1,4 @@
-﻿#pragma checksum "..\..\App.xaml" "{8829d00f-11b8-4213-878b-770e8597ac16}" "2ACA586DE23D1095C00C238C6D9C1B1D99DCE0EDCDEE411679567BBD14A99F1F"
+﻿#pragma checksum "..\..\App.xaml" "{ff1816ec-aa5e-4d10-87f7-6f4963833460}" "68BC721C71289E071597F392E11D572D9C8353B2"
```

Comment:
Delete all the files in in the obj/Debug folder. for some reason the .gitignore isnt working and we dont need these build files. I think the issue will be fixed once #5 is merged